lib/iris/analysis/maths.py

```diff
@@ -220,26 +220,32 @@ def subtract(cube, other, dim=None, ignore=True, in_place=False):
 
     """
     op = operator.isub if in_place else operator.sub
-    return _add_subtract_common(op, 'subtraction', 'subtracted', cube, other,
+    return _add_subtract_common(op, 'subtract', cube, other,
                                 dim=dim, ignore=ignore, in_place=in_place)
 
 
-def _add_subtract_common(operation_function, operation_noun,
-                         operation_past_tense, cube, other, dim=None,
-                         ignore=True, in_place=False):
+def _add_subtract_common(operation_function, operation_name, cube, other,
+                         dim=None, ignore=True, in_place=False):
     """
     Function which shares common code between addition and subtraction
     of cubes.
 
     operation_function   - function which does the operation
                            (e.g. numpy.subtract)
-    operation_symbol     - the textual symbol of the operation (e.g. '-')
-    operation_noun       - the noun of the operation (e.g. 'subtraction')
-    operation_past_tense - the past tense of the operation (e.g. 'subtracted')
+    operation_name       - the public name of the operation (e.g. 'divide')
+    cube                 - the cube whose data is used as the first argument
+                           to `operation_function`
+    other                - the cube, coord, ndarray or number whose data is
+                           used as the second argument
+    dim                  - dimension along which to apply `other` if it's a
+                           coordinate that is not found in `cube`
+    ignore               - ignored! ;-)
```

Comment:
Wow... just discovered that this keyword was already deprecated way back in [version 0.8](https://github.com/rhattersley/iris/blob/61ac271d425eb6037a8e9d3be2458f02f7de111d/lib/iris/analysis/maths.py#L284)!